### ISSUE TITLE
User setPassword cause Object not found again

### ISSUE THREAD

**Issue by flessard** [ParsePlatform/parse-server#342]
Why the field `resetPasswordCode` become empty on save but the `password` cause Object not found error 

```
user.set('resetPasswordCode', '');
user.setPassword(request.params.password);
return user.save(null, { useMasterKey: true });
```

Anyone can help me or can point me how to fix this bug ?
Or what is the best way to update password user in cloud code ? 

if i comment this [line](https://github.com/ParsePlatform/parse-server/blob/master/src/RestWrite.js#L310),  everything Is working perfectly.

**Comment by gfosco** [comment]
This is due to a bug which like #346 will be fixed by #254.  The code is failing trying to delete the session for the user.  The object not found error appears to be from that (and the user change was saved, I believe.)  Will get #254 in soon and we should have a new release version today or tomorrow.

**Comment by flessard** [comment]
@gfosco Nice thanks !

**Comment by flessard** [comment]
@gfosco Bug still present in 2.0.8

**Comment by polo2244** [comment]
@flessard  One Quick question. I used the parse-server-example and deployed to Heroku with the deploy button.
As far as i know, that the parse-server-example has not yet been updated to the latest version (2.0.8).

May i ask how are you updating your parse-server version ?

**Comment by flessard** [comment]
@polo2244 Try to change the version of parse-server in package.json to 2.0.8.

**Comment by flessard** [comment]
fix whit #430
